receiver/carbonreceiver/testdata/config.yaml

```diff
@@ -0,0 +1,34 @@
+receivers:
+  carbon:
+  carbon/allsettings:
+    # endpoint specifies the network interface and port which will receive
+    # Carbon data.
+    endpoint: localhost:8080
+    # transport specifies either "tcp" (the default) or "udp".
+    transport: udp
+    # tcp_idle_timeout is max duration that a tcp connection will idle wait for
+    # new data. This value is ignored is the transport is not "tcp". The default
+    # value is 30 seconds.
+    tcp_idle_timeout: 5s
+    # parser section is used to to configure the actual parser to handle the
+    # received data. The default is "plaintext", see
+    # https://graphite.readthedocs.io/en/latest/feeding-carbon.html#the-plaintext-protocol.
+    parser:
+      # type specifies the type of parser to be used, the default is "plaintext"
+      type: delimiter
+      # config specifies any special configuration of the selected parser.
```

Comment:
Can you also add details about keys that are valid for special configuration of each parser?

Reply:
I will add details and examples when more parser types, for now adding a generic statement.